k8s/nginx-ingress/prod-api-ingress-resource.yaml

```diff
@@ -123,6 +123,22 @@ spec:
             backend:
               serviceName: airqo-calibrate-api-svc
               servicePort: 4001
+          - path: /api/v1/calibrate_tool
+            backend:
+              serviceName: airqo-calibrate-api-svc
+              servicePort: 4001
+          - path: /api/v2/calibrate_tool
+            backend:
+              serviceName: airqo-calibrate-api-svc
+              servicePort: 4001
+          - path: /api/v1/train_calibrate_tool
+            backend:
+              serviceName: airqo-calibrate-api-svc
+              servicePort: 4001
+          - path: /api/v2/train_calibrate_tool
+            backend:
+              serviceName: airqo-calibrate-api-svc
```

Comment:
Are both the calibrate tool and train calibrate tool meant to run the same service or they are essentially both endpoints for accessing this service? @NoahNsimbe

Reply:
Both endpoints are exposed by the service @123MwanjeMike